mesonbuild/build.py

```diff
@@ -2766,6 +2775,12 @@ def get_link_dep_subdirs(self) -> T.AbstractSet[str]:
     def get_all_link_deps(self):
         return []
 
+    def get_rename(self) -> T.List[str]:
+        if self.rename:
+            return self.rename
+        print('get rename for %s' % self.name)
```

Comment:
Debug output

Reply:
oops. I thought I'd cleaned all of those up...